portfolio/src/main/webapp/index.html

```diff
@@ -36,31 +36,31 @@
       <div class="color-buttons">
         <li class="nav-item" id="paint">
           <a class="nav-link white">
-            <i class="material-icons">format_paint</i>
+            <i class="material-icons" style="color: var(--text-primary)"> format_paint</i>
             <span class="link-text">Change color</span>
           </a>
         </li>
         <li class="nav-item color-button">
-          <a class="nav-link coral" onClick="changeAccentColor(this)">
-            <i class="material-icons">stop_circle</i>
+          <a class="nav-link" onClick="changeAccentColor(this)">
+            <i class="material-icons" style="color: var(--coral)">stop_circle</i>
```

Comment:
In general it is recommended use html for structure and css files for presentation. https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

What you had before was ok.